be/src/service/backend_service.h

```diff
@@ -130,18 +125,11 @@ class BackendService : public BackendServiceIf {
     // used for external service, close some context and release resource related with this context
     void close_scanner(TScanCloseResult& result_, const TScanCloseParams& params) override;
 
-    void get_stream_load_record(TStreamLoadRecordResult& result,
-                                const int64_t last_stream_record_time) override;
-
-    void clean_trash() override;
-
-    void check_storage_format(TCheckStorageFormatResult& result) override;
-
-    void ingest_binlog(TIngestBinlogResult& result, const TIngestBinlogRequest& request) override;
-
-    void query_ingest_binlog(TQueryIngestBinlogResult& result,
-                             const TQueryIngestBinlogRequest& request) override;
-
+    // TODO(AlexYue): The below cloud backend functions should be implemented in 
+    // CloudBackendService
+    ////////////////////////////////////////////////////////////////////////////
+    // begin cloud backend functions
+    ////////////////////////////////////////////////////////////////////////////
     void pre_cache_async(TPreCacheAsyncResponse& response,
```

Comment:
这个将来非存算分离会用到不

Reply:
They will only serve for Cloud-Mode, currently they have to be inherited and implemented because `BackendServiceIf` is one interface with only pure functions. These cloud backend functions should be moved to cloud backend service afterwards.